linux_os/guide/system/permissions/restrictions/sysctl_user_max_user_namespaces/rule.yml

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Disable the use of user namespaces'
+
+description: |-
+    {{{ describe_sysctl_option_value(sysctl="user.max_user_namespaces", value="0") }}}
+    When containers are deployed on the machine, the value should be set
+    to large non-zero value.
+
+rationale: |-
+    User namespaces are used primarily for Linux containers. The value 0
+    effectively disallows the use of user namespaces.
+
```

Comment:
```suggestion
warnings:
    - general: |-
       This configuration baseline was created to deploy the base operating system for general purpose
       workloads. When the operating system is configured for certain purposes, such as to host Linux Containers,
       it is expected that <tt>user.max_user_namespaces</tt> will be enabled.
```

Reply:
Lets give administrators and auditors an understanding of when to expect user.max_user_namespaces to be enabled.

Reply:
Added in d518ade748d0ca02854f57d51f6f3174d4394823.